example/android/build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:7.3.1")
+        classpath('com.android.tools.build:gradle:7.4.2')
```

Comment:
Do we need this upgrade in this PR? Otherwise move to separate smaller PR for clarity, or document why it is needed.